.check-style-ignore

```diff
@@ -0,0 +1,7 @@
+# File paths matching the regular expressions in this file (one per line)
+# will be skipped by data/check-style. Empty lines and lines starting with
+# a hash mark are ignored.
+
+platform/common/xdp-parent-private\.h$
+platform/drm/kms\.[hc]$
+platform/wayland/os-compatibility\.[hc]$
```

Comment:
What motivates the exclusion of these files?

Reply:
These files contain utility code imported from other projects as-is, without modifications. The other projects may not follow the same conventions, so they result in false `check-style` positives (see #606 for an example).